### PR TITLE
[15.0][MIG] sale_attached_product: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernesto-garcia-tecnativa**
@tecnativa TT41762

### CODE REVIEW

File: sale_attached_product/README.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure attached products:
+
+#. Go to *Sales > Catalog > Products* and choose on you want to attach products to.
```
Comment (DucTruongKomit) [inline sale_attached_product/README.rst:41]: I cannot see the menu Catalog? [sale_attached_product_instance url](http://oca-sale-workflow-15-0-pr2393-941fc466917e.runboat.odoo-community.org/web?debug=1#cids=1&menu_id=331&action=505&model=sale.order&view_type=list)

image

Do I do something wrong?

Comment (ernesto-garcia-tecnativa) [reply sale_attached_product/README.rst:41]: I update the CONFIGURE.rst file

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza @victoralmau please review

**Review by victoralmau — APPROVED** [review]
Code review OK

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Merging after fixing CI in #2405 

/ocabot merge nobump

**Pull request merged**